src/status_im/ui/screens/activity_center/views.cljs

```diff
@@ -0,0 +1,151 @@
+(ns status-im.ui.screens.activity-center.views
```

Comment:
rollback pls

Reply:
still not addressed

Reply:
this file has been deleted, you need to rollback it and remove

Reply:
cc @ilmotta

Reply:
Right @flexsurfer, this file is long gone.

Reply:
@flexsurfer  i missed your comment before. I have addressed that now